src/features/import/l10n/messagesIds.ts

```diff
@@ -0,0 +1,16 @@
+import { m, makeMessages } from 'core/i18n';
+
+export default makeMessages('feat.import', {
+  importDialog: {
```

Comment:
This will be huge (when all of Ziggi's and Haeju's work is included), so should maybe be split into multiple sub-keys?